python/lsst/summit/utils/astrometry/plotting.py

```diff
@@ -93,46 +93,49 @@ def plot(exp, icSrc=None, filteredSources=None, saveAs=None,
         plt.plot(filteredSources['base_SdssCentroid_x'],
                  filteredSources['base_SdssCentroid_y'],
                  **markerStyle)
-    plt.arrow(compassCenter[0],
-              compassCenter[1],
-              -compassSize*np.sin(rotpa),
-              compassSize*np.cos(rotpa),
-              color='green', width=20)
-    plt.text(compassCenter[0] - textDistance*np.sin(rotpa),
-             compassCenter[1] + textDistance*np.cos(rotpa),
-             'N',
-             color='green', fontsize=fontsize, weight='bold')
-    plt.arrow(compassCenter[0],
-              compassCenter[1],
-              compassSize*np.cos(rotpa),
-              compassSize*np.sin(rotpa),
-              color='green', width=20)
-    plt.text(compassCenter[0] + textDistance*np.cos(rotpa),
-             compassCenter[1] + textDistance*np.sin(rotpa),
-             'E',
-             color='green', fontsize=fontsize, weight='bold')
+
+    if not np.isnan(rotpa):  # need a rotation angle for the compass
```

Comment:
Incredibly, I have a comment on a two-line PR... does it make sense for this and the check below to be `if np.isinfinite(rotpa)` instead? I suppose you don't need to check if it's between 0 and 2pi.

Reply:
Errr, yeah, I guess. Honestly, I'd just never contemplated that they could be +/- inf, but I guess it doesn't hurt to use `isfinite` instead.